analyzers/tests/SonarAnalyzer.Test/Rules/UriShouldNotBeHardcodedTest.cs

```diff
@@ -21,7 +21,7 @@
 using CS = SonarAnalyzer.Rules.CSharp;
 using VB = SonarAnalyzer.Rules.VisualBasic;
 
-namespace SonarAnalyzer.UnitTest.Rules
+namespace SonarAnalyzer.Test.Rules
```

Comment:
Please make this also file scoped namespace (also in another PR to not block this one).

Reply:
There are 1686 files that would need this - that's too big PR that would be impossible to review. So I'll leave this on CaYC over time.